content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Change Notes
+description: Istio 1.8 release notes.
+weight: 10
+---
+
+# Release Notes
+
+## Traffic Management
+
+- **Added** support for injecting istio-cni into `k8s.v1.cni.cncf.io/networks` annotation with pre-existing value that uses JSON notation.
+  ([Issue #25744](https://github.com/istio/istio/issues/25744))
+
+- **Added** support for `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` insert operations for `HTTP_ROUTE` in EnvoyFilter  ([Issue #26692](https://github.com/istio/istio/issues/26692))
+
+- **Added** `REPLACE` operation for EnvoyFilter. `REPLACE` operation can replace the contents of a named filter with new contents. It is only valid for `HTTP_FILTER` and `NETWORK_FILTER`.
+  ([Issue #27425](https://github.com/istio/istio/issues/27425))
+
+- **Added** Istio resource status now includes Observed Generation
```

Comment:
Not sure we need to capitalize "observed generation"

Reply:
done